codecov.yml

```
@@ -0,0 +1,12 @@
+coverage:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)